cyences_app_for_splunk/bin/cs_product_list.py

```diff
@@ -320,28 +320,84 @@ def build_source_reviewer_search(by, values):
         "macro_configurations": [
             {
                 "macro_name": "cs_windows_idx",
                 "label": "Windows Data",
-                "search": r"""`cs_windows_idx` | stats count | eval label="Windows" 
+                "search": r"""`cs_windows_idx` NOT source IN ("WinEventLog:DFS Replication","WinEventLog:Directory Service","WinEventLog:Microsoft-AzureADPasswordProtection-DCAgent/Admin","PerfmonMk:DFS_Replicated_Folders","WinEventLog:DNS Server") NOT sourcetype IN ("MSAD:NT6:Netlogon","MSAD:NT6:Replication","MSAD:NT6:Health","MSAD:NT6:SiteInfo","Script:TimesyncConfiguration","Script:TimesyncStatus","windows:certstore:ca:issued","MSAD:NT6:DNS","MSAD:NT6:DNS-Health","MSAD:NT6:DNS-Zone-Information","PerfmonMk:DNS") | stats count | eval label="Windows" 
 | append [| search `cs_windows_idx` sourcetype="*WinEventLog" source="*WinEventLog:Security" | stats count | eval label="Windows Security", search="sourcetype=\"*WinEventLog\" source=\"*WinEventLog:Security\""] 
 | append [| search `cs_windows_idx` sourcetype="*WinEventLog" source="*WinEventLog:System" | stats count | eval label="Windows System", search="sourcetype=\"*WinEventLog\" source=\"*WinEventLog:System\""] 
 | append [| search `cs_windows_idx` sourcetype="ActiveDirectory" | stats count | eval label="Windows AD", search="sourcetype=\"ActiveDirectory\""] 
-| append [| search `cs_windows_idx` source=powershell sourcetype="MSAD:*:Health" | stats count | eval label="Windows AD Health", search="source=powershell sourcetype=\"MSAD:*:Health\""] 
 | table label search count""",
-                "host_reviewer_search": '| tstats count where `cs_windows_idx` by source sourcetype host | eval sources = source." (".sourcetype.")" | table sources host count',
-                "sources_reviewer_search": r"""| tstats values(host) as hosts where `cs_windows_idx` by source sourcetype index | eval sources = source." (".sourcetype.")" | stats count values(index) as index dc(hosts) as host_count by sources | eval sources=if(count>0,sources,"`cs_windows_idx`") | append [
+                "host_reviewer_search": '| tstats count where ((`cs_windows_idx`) OR (index=* sourcetype IN ("*WinEventLog","ActiveDirectory") OR source IN ("*WinEventLog:Security","*WinEventLog:System","powershell*"))) NOT source IN ("WinEventLog:DFS Replication","WinEventLog:Directory Service","WinEventLog:Microsoft-AzureADPasswordProtection-DCAgent/Admin","PerfmonMk:DFS_Replicated_Folders","WinEventLog:DNS Server") NOT sourcetype IN ("MSAD:NT6:Netlogon","MSAD:NT6:Replication","MSAD:NT6:Health","MSAD:NT6:SiteInfo","Script:TimesyncConfiguration","Script:TimesyncStatus","windows:certstore:ca:issued","MSAD:NT6:DNS","MSAD:NT6:DNS-Health","MSAD:NT6:DNS-Zone-Information","PerfmonMk:DNS") by source sourcetype host | eval sources = if(sourcetype IN ("WinEventLog","WinHostMon","exec","XmlWinEventLog*","powershell"),source,sourcetype) | stats sum(count) as count by sources host',
```

Comment:
Please remove AD and DNS related stuff from here.

Reply:
It's already excluded at the end with NOT source IN (....) NOT sourcetype IN (....)